themes/wilson_theme_starterkit/src/js/components/statistics.js

```diff
@@ -0,0 +1,53 @@
+// Import the countUp.js module.
+import { CountUp } from '../libraries/countUp.min.js';
```

Comment:
I noticed that it had been done this way on Wave, and got confused, because I was expecting it to be included in the `libraries.yml` file. 

I suspect it _should_ also be added to the library definition, because the `statistics.js` file _can't run_ unless the `countUp` library is there on the page as well. That might then mean you don't need to import it this way.

Not sure if @rob-howells or @mangyfox has a better idea/preference though.

Reply:
I think if it was me I would add it as a library and then mark it as a dependency on the statistics library. But that's just me. I don't think I have seen scripts imported this way in Drupal scripts before.

Reply:
I would have liked to include it in the traditional library include way but CountUp plain refuses to work that way. They're trying to be all cool and modern and use Javascript modules. Maybe I was doing something wrong, or we need to find a more vanilla/old school library if we want to do it the Drupal way. I do want to avoid jQuery libraries though.

Reply:
Did you try using the UMD version of CountUp? https://github.com/inorganik/CountUp.js#umd-module The docs suggest this is there to support legacy browsers that don't support ES modules, so may work better with Drupal's library system